tests/count_from_field_fn_deep_test.js

```diff
@@ -1,28 +1,28 @@
 if (Meteor.isServer) {
-  Meteor.publish('count_from_field_fn_deep', function (testId) {
-    Counts.publish(this, 'posts' + testId, Posts.find({testId: testId}),
+  Meteor.publish('count_from_field_fn_deep', async function (testId) {
+    await Counts.publish(this, 'posts' + testId, Posts.find({testId: testId}),
         {countFromField: function (doc) { return doc.a.b; }});
   });
 
   Meteor.methods({
-    setup_deep_countFromField_fn: function (testId) {
-      H.insert(testId, 0, {a: {b: 2}});
-      H.insert(testId, 1, {a: {b: 3}});
+    setup_deep_countFromField_fn: async function (testId) {
+      await H.insert(testId, 0, {a: {b: 2}});
```

Comment:
Shouldn't this be `H.insertAsync`?

Reply:
Nope. I know it looks confusing, but `H` is the helper module in `helper.js`, where there is something like this:
```js
this.H.insertFactory = function insertFactory (collection) {
  // return a DRY function that inserts docs into one collection.
  return function insert (testId, docNum, doc) {
    var testDoc = _.extend({}, doc, {
      _id:    H.docId(testId, docNum),
      testId: testId,
    });
    return collection.insertAsync(testDoc);
  };
}

// ...

this.H.insert = this.H.insertFactory(Posts);
```